### DISCUSSION TITLE
Ability to set a frame offset for TileMap animations

### DISCUSSION THREAD

**Discussion by PilotIRL** [godotengine/godot-proposals#10502] [2D]
I'm working on converting a java 2D game to godot. The game has user contributed maps that includes animated tiles. The animations have the option of an offset that specifies the starting frame in the animation. In this way users can have the same animation tiles but with a time shift, which creates unique visual effects on the map.

It would be nice if there was a setting for animation frame offset that could be set per cell on the TileMapLayer (default 0)

**Comment by AThousandShips** [comment]
Thank you for your proposal, consolidating in:
* https://github.com/godotengine/godot-proposals/issues/7461
